meteor/sofie-server/model/entities/rundown.ts

```diff
@@ -27,11 +38,33 @@ export class Rundown extends BasicRundown {
 	private nextSegment: Segment
 	private nextPart: Part
 
+	private previousPart?: Part
+
 	private infinitePieces: Map<string, Piece> = new Map()
 
 	constructor(rundown: RundownInterface) {
 		super(rundown.id, rundown.name, rundown.isRundownActive, rundown.modifiedAt)
 		this.segments = rundown.segments ?? []
+
+		if (rundown.alreadyActiveProperties) {
```

Comment:
Inverting this into a guard would improve readability, in my mind.

Reply:
We are going to do something about this one.
Anders has some ideas of maybe using a Cursor or something similar and that would make this go away. Leaving it as is for now.